plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,627 @@
+ Generally user "munin" is ok. Then, execute "sudo su munin -s /bin/bash", 
+ "ssh-keygen" and "ssh-copy-id" to both Control Stations with newly created 
+ user.
+ 
+ Make a link from /usr/share/munin/plugins/emc_vnx_file_stats to 
+ /etc/munin/plugins/. If you want to get NFS statistics, name the link as 
+ "emc_vnx_file_nfs_stats_<NAME>", otherwise to get Basic Datamover statistics
+ you have to name it "emc_vnx_file_basicdm_stats_<NAME>", where <NAME> is any
+ arbitrary name of your storage system. The plugin will return <NAME> in its 
+ answer as "host_name" field.
+
+ For example, assume your storage system is called "VNX5300".
+ Make a configuration file at 
+ /etc/munin/plugin-conf.d/emc_vnx_file_stats_VNX5300
+ 
+ [emc_vnx_file_*]
+ user munin				
+ env.username operator1				
+ env.cs_addr 192.168.1.1 192.168.1.2		
+ env.nas_servers server_2 server_3		
+
+ Where: 
+ user - SSH Client local user
+ env.username - Remote user with Operator role
+ env.cs_addr - Control Stations addresses
+ env.nas_servers - This is the default value and can be omitted
+
+=head1 HISTORY
+
+ 08.11.2016 - First Release
+ 17.11.2016 - NFSv4 support, Memory section
+ 16.12.2016 - Merged "NFS" and "Datamover Stats" plugins
+ 26.12.2016 - Compatibility with Munin coding style
+
+=cut
+
+export LANG=C
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+ssh_check_cmd() {
+	ssh -q $username@$1 "/nasmcd/sbin/getreason | grep -w slot_\`/nasmcd/sbin/t2slot\` | cut -d- -f1"
+}
+
+check_conf () {
+	if [ -z "$username" ]; then
+		echo "No username ('username' environment variable)!"
+		return 1
+	fi
+
+	if [ -z "$cs_addr" ]; then
+		echo "No control station addresses ('cs_addr' environment variable)!"
+		return 1
+	fi
+
+	#Choosing Cotrol Station. Code have to be "10"
+	for CS in $cs_addr; do
+		if [[ "10" -eq "$(ssh_check_cmd $CS)" ]]; then
+			PRIMARY_CS=$CS
+			break
+		fi
+	done
+
+	if [ -z "$PRIMARY_CS" ]; then
+		echo "No alive primary Control Station from list \"$cs_addr\"";
+		return 1
+	fi
+	return 0
+}
+
+if [ "$1" = "autoconf" ]; then
+	check_conf_ans=$(check_conf)
+	if [ $? -eq 0 ]; then
+		echo "yes"
+	else
+		echo "no ($check_conf_ans)"
+	fi
+	exit 0
+fi
+
+if [ "$1" = "suggest" ]; then
+	echo "nfs_stats"
+	echo "basicdm_stats"
+	exit 0;
+fi
+
+STATSTYPE=$(echo "${0##*/}" | cut -d _ -f 1-5)
+if [ "$STATSTYPE" = "emc_vnx_file_nfs_stats" ]; then STATSTYPE=NFS; 
+elif [ "$STATSTYPE" = "emc_vnx_file_basicdm_stats" ]; then STATSTYPE=BASICDM;
+else echo "Do not know what to do. Name the plugin as 'emc_vnx_file_nfs_stats_<HOSTNAME>' or 'emc_vnx_file_basicdm_stats_<HOSTNAME>'"; exit 1; fi
+# echo $STATSTYPE
+
+TARGET=$(echo "${0##*/}" | cut -d _ -f 6)
+: ${nas_servers:="server_2 server_3"}
+
+check_conf 1>&2 || exit 1
+
+run_remote () {
+	ssh -q "$username@$PRIMARY_CS" ". /home/""$username""/.bash_profile; $*"
+}
+
+echo "host_name ${TARGET}"
+
+if [ "$1" = "config" ] ; then
+	for server in $nas_servers; do
+		run_remote nas_server -i $server | grep -q 'type *= nas'
```

Comment:
Please quote `$server` in order to avoid shellcheck warnings (even though there cannot be a problem here).

Additionally you can squash the if-continue statement below by just adding the suffix `| continue` to this line.